docs/guide/core-widgets.md

```diff
@@ -88,6 +88,29 @@ You can use the same tag in several styles with various CSS classes.
 Including a class with a style will not automatically apply any styles. You still need to [write your own CSS](/guide/front-end-assets.md) for the class.
 :::
 
+While most tags that are not used by the formatting tools need to be added to the list of allowed tags in the widget before use, the `<span>` tag is allowed by default and useful for styling blocks of text. Selecting a block of text, even with spaces, and applying a style using the span tag will surround that block of text with span tags with class attribute(s). For example, you can highlight just a portion of a paragraph.
```

Comment:
"While most tags that are not used need to be added before use..."

Huh? What list of allowed tags? The rich text widget determines which tags are allowed automatically based on the configuration of "styles" and the presence/absence of other controls. Um, right?

(See `toolbarToAllowedTags` in the source code)

Reply:
I suggest avoiding the word "block" here. The point we're trying to make is that if the tag is "span" then you can apply a style to any selection of text, it doesn't toggle the entire surrounding block element to a different element.

It looks like `span` isn't the only tag that allows this. The full list is here:

```
        'b', 'strong', 'code', 'mark', 'em', 'i',
        'a', 's', 'del', 'strike', 'span', 'u', 'anchor',
        'superscript', 'subscript'
```

But I wouldn't list the ones that already have a matching toolbar control of their own.

Reply:
> "While most tags that are not used need to be added before use..."
> 
> Huh? What list of allowed tags? The rich text widget determines which tags are allowed automatically based on the configuration of "styles" and the presence/absence of other controls. Um, right?
> 
> (See `toolbarToAllowedTags` in the source code)

This was an unfortunate choice of words. While `toolbarToAllowedTags` dynamically allows any tag, that doesn't mean that Tiptap knows what to do with that tag. If you try to add something like:
```js
styles: [
  {
    tag: 'abbr',
    label: 'Abbreviation'
  }
]
```
Tiptap will just choke. It doesn't know if it is a node, or a mark. I've tried rephrasing.

Reply:
> I suggest avoiding the word "block" here. The point we're trying to make is that if the tag is "span" then you can apply a style to any selection of text, it doesn't toggle the entire surrounding block element to a different element.
> 
> It looks like `span` isn't the only tag that allows this. The full list is here:
> 
> ```
>         'b', 'strong', 'code', 'mark', 'em', 'i',
>         'a', 's', 'del', 'strike', 'span', 'u', 'anchor',
>         'superscript', 'subscript'
> ```
> 
> But I wouldn't list the ones that already have a matching toolbar control of their own.

I've added them all in with the additional info that they can be added to the `styles` dropdown in order to add a certain class. I only included them all because only the `mark` and `u` didn't have their own button. I can remove this section.